src/Http/FirebaseShortener.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace LaraCrafts\UrlShortener\Http;
+
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\Psr7\Request;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Str;
+use Psr\Http\Message\ResponseInterface;
+use function GuzzleHttp\json_decode;
+
+class FirebaseShortener extends RemoteShortener
+{
+    private static $SUFFIXES = ['UNGUESSABLE', 'SHORT'];
+
+    protected $client;
+    protected $defaults;
+
+    /**
+     * Create a new Shorte.st shortener.
+     *
+     * @param \GuzzleHttp\ClientInterface $client
+     * @param string $token
+     * @param string $domain
+     * @param string $suffix
+     */
+    public function __construct(ClientInterface $client, string $token, string $domain, string $suffix)
+    {
+        if (!Arr::has(self::$SUFFIXES, Str::upper($suffix))) {
+            $suffix = self::$SUFFIXES[0];
```

Comment:
I think you might want to throw in this case

Reply:
In the README I stated the following:

> The suffix have the value of `SHORT` or `UNGUESSABLE`. If other values are set the library will use the `UNGUESSABLE` value.

So lets decide if this should be an exception or use a fallback.

Reply:
In my opinion it should throw, because every other used variable that's missing or wrong wil cause an error as well, this would be the only exception.

@yak0d3 thoughts?